controls/autocomplete/features/autocomplete-key-features.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Key Features
+page_title: Key Features
+position: 0
+slug: autocomplete-key-features
+---
+
+# Key Features
+
+The purpose of this help article is to show you the key features of the **RadAutoComplete** control.
+
+## Tokens Support
+
+With AutoComplete you could enable users to search for and select several items. These items appear as tokens that can easily be deselected using their close button. 
+
+**DisplayMode** property determines whether a single or multiple selection is enabled.  The default DisplayMode is “Plain”, for multiple selection you would need to set it to “Tokens”.
+
+>tip Tokens support is available only for Android and iOS platforms.
+
+<snippet id='autocomplete-features-tokenssupport'/>
+
+#### Figure 1: Tokens Support
+
+![](../images/autocomplete-features-tokens.png)
+
+## Filtering Options
+
+You can determine the filtering behavior of RadAutoComplete through the **CompletionMode** property. The available completion modes are "StartsWith" (default)  and  "Contains".
+
+>tip In data-binding scenarios you will also need to set **TextSearchPath** property, which defines the name of the property the search function will be executed against. For more details check the [Data Binding]({% slug autocomplete-data-binding %}) topic.
+
+<snippet id='autocomplete-features-filter'/>
+
+#### Figure 2: Filtering options
+
+![](../images/autocomplete-features-filtering.png)
+
+## Watermark
+
+RadAutoComplete exposes **Watermark** property used to give guidance to the end user on what should be entered in the text input. The watermark text is displayed when the control is empty.
+
+<snippet id='autocomplete-features-watermark'/>
+
+#### Figure 3: Watermark
+
+![](../images/autocomplete-features-watermark.png)
+
+## Clear Button Visibility
+
+Clear button, which appears at the right side of the input field when the AutoComplete is in focus, gives the end-user the option to quickly clear the entered values. You could control the visibility of the button through the **IsClearButtonVisible** property.
+
+## NoResults Message
+
+NoResults message appears in the popup used for the list of suggestions whenever the control cannot find any matching items.  You could easily customize it through the **NoResultsMessage** property.
+
+<snippet id='autocomplete-features-noresultsmessage'/>
+
+#### Figure 3: NoResultsMessage
+
+![](../images/autocomplete-features-noresults.png)
+
+## Search Threshold
+
+By default the search is triggered as soon as the user types into the input field. Using **SearchThreshold** you could configure AutoComplete to trigger the search after a certain number of letters is entered. 
```

Comment:
**By** using "SearchTreshold", you **can** (I think can is preferable over could in how-to scenarios, but probably both are used)